app/controllers/application_controller.rb

```diff
@@ -1,6 +1,8 @@
 class ApplicationController < ActionController::Base
   default_form_builder GOVUKDesignSystemFormBuilder::FormBuilder
 
+  include DfE::Analytics::Requests
```

Comment:
Worth adding a quick request spec to cover this? something like:

```
it "sends DFE Analytics web request event" do
  expect { get root_path }.to have_sent_analytics_event_types(:web_request)
end
```

Reply:
could we also add this in the `API::BaseController` to cover API requests as well?